### ISSUE TITLE
Can the FCM implemented by microG wake up apps that have been killed?

### ISSUE THREAD

**Issue by daiaji** [microg/GmsCore#1925]
https://github.com/kooritea/fcmfix
This exposed module allows Google's native FCM to wake up killed apps.
Does the FCM implemented by microG also have similar functions?

**Comment by illerokcob** [comment]
Yes, force-stopped applications are woken up by microG when the app receives a cloud message through GCM.

**Comment by ale5000-git** [comment]
Obviously it works only if microG GmsCore is installed as privileged application.

**Comment by daiaji** [comment]
> Obviously it works only if microG GmsCore is installed as privileged application.

I've installed microG to the /system/priv-app folder using magisk, is that enough？
